From 16941c47a6f0fc2f1679725d55cde221f3c3a6ef Mon Sep 17 00:00:00 2001
From: Sumit Bose <sbose@redhat.com>
Date: Fri, 7 Sep 2018 22:12:02 +0200
Subject: [PATCH 32/83] intg: user default locale

Some checks depend on english error messages so checks should be always
run with the default locale.

Related to https://pagure.io/SSSD/sssd/issue/3500

Reviewed-by: Jakub Hrozek <jhrozek@redhat.com>
---
 src/tests/intg/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/tests/intg/Makefile.am b/src/tests/intg/Makefile.am
index 65da9ca..6f7605b 100644
--- a/src/tests/intg/Makefile.am
+++ b/src/tests/intg/Makefile.am
@@ -126,6 +126,7 @@ intgcheck-installed: config.py passwd group
 	PATH="$$(dirname -- $(SLAPD)):$$PATH" \
 	PATH="$(DESTDIR)$(sbindir):$(DESTDIR)$(bindir):$$PATH" \
 	PATH="$$PATH:$(abs_builddir):$(abs_srcdir)" \
+	LANG=C \
 	PYTHONPATH="$(abs_builddir):$(abs_srcdir)" \
 	LDB_MODULES_PATH="$(DESTDIR)$(ldblibdir)" \
 	NON_WRAPPED_UID=$$(id -u) \
-- 
2.9.5

